app/Http/Controllers/Bible/BiblesController.php

```diff
@@ -188,13 +188,14 @@ public function index()
             )->orderByRaw($order_by)->groupBy('bibles.id');
 
 
-            $bibles = $bibles->paginate($limit);
-            $bibles_return = fractal(
-                $bibles->getCollection(),
-                BibleTransformer::class,
-                new DataArraySerializer()
-            );
-            return $bibles_return->paginateWith(new IlluminatePaginatorAdapter($bibles));
+            // $bibles = $bibles->paginate($limit);
+            // $bibles_return = fractal(
+            //     $bibles->getCollection(),
+            //     BibleTransformer::class,
+            //     new DataArraySerializer()
+            // );
+            // return $bibles_return->paginateWith(new IlluminatePaginatorAdapter($bibles));
+            return $bibles->first();
         });
 
         return $this->reply($bibles);
```

Comment:
re: cache size log, I was thinking we would need to log the size of the SQL response, going into cache. Our thought is that the size is too big and the cache rejects it